airflow/providers/amazon/aws/operators/redshift_data.py

```diff
@@ -87,6 +90,7 @@ def __init__(
         aws_conn_id: str = "aws_default",
         region: str | None = None,
         workgroup_name: str | None = None,
+        deferrable: bool = conf.getboolean("operators", "default_deferrable", fallback=False),
```

Comment:
can't we use the wait_for_completion parameter alone to deduce the deferrablity?

Reply:
They are different things. When `wait_for_completion` is true, we'll wait in worker. On the other hand, when `deferrable=True`, the checking behavior happens in triggerer